messages/lightning.preview.app.md

```diff
@@ -39,6 +39,106 @@ For mobile virtual devices, specify the device ID to preview. If omitted, the fi
 
 Unable to determine App Id for %s
 
+# error.device.notfound
+
+Unable to find device %s
+
+# spinner.device.boot
+
+Booting device %s
+
+# spinner.cert.gen
+
+Generating self-signed certificate
+
+# spinner.extract.archive
+
+Extracting archive
+
+# spinner.download.preparing
+
+Preparing to download
+
+# spinner.downloading
+
+Downloading
+
+# certificate.attention
+
+╔═══════════╗
+║ ATTENTION ║
+╚═══════════╝
+
+# certificate.installation.description
+
+If you have not done so already, please install the self-signed certificate on your device before proceeding. The certificate file is located at
+
+`%s`
+
+Follow the steps below to install the certificate:
+
+%s
+
+# certificate.installation.steps.ios
+
+1. Drag and drop the file on to your booted simulator.
```

Comment:
For both platforms, if it doesn't blow out the scope I wonder if we should consider more concise "inline" instructions, providing links to markdown pages on our repo that describe the installation instructions, complete with images? Maybe that could be a stretch goal for beta, or a GA consideration. I'm afraid anything less is likely going to result in confusion for our users.

Reply:
I wanted to do something like that at first but on Android the steps for navigating through the Settings app is different from one API to another. So I wasn't sure how that would work out if we wanted to use markdown pages containing screenshots that would be different depending on the emulator API level.

But if we go with printing the instructions to console like I'm doing in this PR, at least then I can programmatically  determine what is the API level for the user specified emulator and I can dynamically compose the steps.

But in general I think we should try to see if we can find a solution where the user would not need to install the cert manually (if at all possible)